tools/lib/NQP/Config/Rakudo.pm

```diff
@@ -272,13 +272,34 @@ sub configure_moar_backend {
             moar => "Unable to read configuration from NQP on MoarVM" );
     }
 
+    # Relocatability is not supported on AIX.
+    $self->{options}->{"no-relocatable"} ||= !!( $^O =~ /^(?:aix|openbsd)$/ );
+    if ( $self->{options}->{"no-relocatable"} ) {
+        $config->{relocatable} = 0;
```

Comment:
I have a recommendation here. Whatever is in the config will likely end up expanded in a template. Thus it will become exposed to a user. From this point of view some meaningful values would be pretty thing to have.

Reply:
0 = false, 1 = true is a widely known convention for flags. What values would you propose that are more meaningful?

Reply:
It is but when used internally. And even in this case it is preferable to hide them under some constants or aliases. What I would do here is set `relocatable` to _reloc_ or _nonreloc_. Quite readable and it would have another purpose. Where you generate `nqp-m` from a template  you could replace your whole `@perl()@` block with just one line where template name is defined with: 

`@shquot(@template(@platform@/nqp-m-@relocatable@)@)@`

Then you'd just have two templates: `nqp-m-reloc` and `nqp-m-nonreloc`. The advantage is that a 3rd party user would understand the difference at a single glance at the template names.

Reply:
Changed as proposed. Thanks for the idea!